Snakemake/workflow/Snakefile

```diff
@@ -39,6 +43,9 @@ vcfdir=config['vcfDir'] # from config file, the path to vcf
 Project = config['PROJECT']
 mapdir = config['genmap']
 
+# if want to split multiallelic sites set to TRUE:
```

Comment:
What does this mean?

Reply:
I am testing something. I've updated now and comment that out

Reply:
@gmafrafortuna there is still some "HEAD" lines in here

Reply:
fixed